Source/Core/DolphinWX/ControllerConfigDiag.cpp

```diff
@@ -26,26 +26,24 @@
 #include "Core/HW/GCKeyboard.h"
 #include "Core/HW/GCPad.h"
 #include "Core/HW/SI.h"
-#if defined(__LIBUSB__) || defined (_WIN32)
-#include "Core/HW/SI_GCAdapter.h"
-#endif
 #include "Core/HW/Wiimote.h"
 #include "Core/HW/WiimoteReal/WiimoteReal.h"
 #include "DolphinWX/ControllerConfigDiag.h"
 #include "DolphinWX/InputConfigDiag.h"
+#include "DolphinWX/Config/GCAdapterConfigDiag.h"
+#include "InputCommon/GCAdapter.h"
 
 #if defined(HAVE_XRANDR) && HAVE_XRANDR
 #include "DolphinWX/X11Utils.h"
 #endif
 
-wxDEFINE_EVENT(wxEVT_ADAPTER_UPDATE, wxCommandEvent);
-
 ControllerConfigDiag::ControllerConfigDiag(wxWindow* const parent)
 	: wxDialog(parent, wxID_ANY, _("Dolphin Controller Configuration"))
 {
 	m_gc_pad_type_strs = {{
 		_("None"),
 		_("Standard Controller"),
+		_("Wii U Gamecube Adapter"),
```

Comment:
GameCube must be spelled with a capital C.

Also, it might be clearer in which way Wii U relates to GameCube if this says "GameCube Controller Adapter for Wii U". Is that too long?